ksql-functional-tests/src/test/resources/test-runner/correct/udf/statements.sql

```diff
@@ -0,0 +1,2 @@
+CREATE STREAM TEST (ID bigint, NAME varchar, VALUE double) WITH (kafka_topic='test_topic', value_format='DELIMITED', key='ID');
+CREATE STREAM S1 as SELECT myudf(name) FROM test;
```

Comment:
how was `test-udf-1.0-SNAPSHOT.jar` created? It's nice to be able to update this jar if necessary

Reply:
We already have test UDFS in the code base, right?   We should be either using those, or adding a new test-udf module.  We shouldn't be checking in such jars as binaries.